helm/cluster-cloud-director/values.yaml

```diff
@@ -93,11 +93,15 @@ nodePools: {}
   #   class: default  # Node classes name (defined above)
   #   replicas: 2
 
-ssh:
-  users:  # (Optional) public SSH keys/users to access the nodes.
-    - name: ""
-      authorizedKeys:
-        - ""
+sshTrustedUserCAKeys:
+  # Taken from https://vault.operations.giantswarm.io/v1/ssh/public_key
+  - ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIM4cvZ01fLmO9cJbWUj7sfF+NhECgy+Cl0bazSrZX7sU vault-ca@vault.operations.giantswarm.io
+
+osUsers:
```

Comment:
I removed this from ssh section since it is not bound to ssh context only. Without sshAuthorizedKeys, this can be used too.